templates/etc/sudoers.d/ansible.j2

```diff
@@ -1,6 +1,6 @@
-#{{ ansible_managed }}
-
-# Ansible-managed users with NOPASSWD
+###                                     ###
+### This file is controlled by Ansible. ###
+###                                     ###
```

Comment:
Not if we define `ansible_managed` with static content instead of 
`ansible_managed = Ansible managed: {file} modified on %Y-%m-%d %H:%M:%S by {uid} on {host}`